rasterio/_crs.pyx

```diff
@@ -375,7 +375,16 @@ cdef class _CRS:
             return _CRS.from_epsg(epsg_code)
 
         # Continue with the general case.
-        proj = ' '.join(['+{}={}'.format(key, val) for key, val in data.items()])
+        pjargs = []
+        for key, val in data.items():
+            if val is None or str(val) == 'True':
```

Comment:
@drnextgis what would you think about tightening this a little more and going with the following?

```suggestion
            if val is None or val is True:
```

Reply:
`is True` caused pyproj problems with numpy bool: https://github.com/pyproj4/pyproj/pull/590. I would recommend checking for numpy bool as well if you don't want the string check.

Reply:
@snowman2 why do you think rasterio should support numpy bool here?

Reply:
Just a recommendation based on an issue someone raised previously for pyproj: https://github.com/pyproj4/pyproj/issues/564